### PR TITLE
Better title for README.

### PR DESCRIPTION
**Pull request by mhrmm**
@jiljames I made a more readable README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# thesis-jillian-james
\ No newline at end of file
+# Jillian's Thesis Project
```
Comment (jiljames) [inline README.md]: No, I have three Ls in my name!

**Review by jiljames — COMMENTED** [review]
Done with pass. PTAL.

**Comment by mhrmm** [comment]
All fixed. PTAL.

**Comment by jiljames** [comment]
ok. LGTM.

**Pull request closed without merging**